Template `npm start` fails because of missing dependencies

After creating a project, when `npm start` is ran, it errors out with the following:

```
ERROR in ./node_modules/@hitachivantara/uikit-react-core/dist/modern/Chart/Plot/Plot.js 6:0-42
Module not found: Error: Can't resolve 'plotly.js-basic-dist' in './node_modules/@hitachivantara/uikit-react-core/dist/modern/Chart/Plot'
resolve 'plotly.js-basic-dist' in './node_modules/@hitachivantara/uikit-react-core/dist/modern/Chart/Plot'
[...]
```

Furthermore, when I run `npm run lint` or `npm run format`, a similar error occurs because `prettier` isn't installed:

```
Oops! Something went wrong! :(

ESLint: 8.7.0

Error: Cannot find module 'prettier'
```

--- a/template.json
+++ b/template.json
@@ -13,6 +13,7 @@
       "i18next-browser-languagedetector": "^6.1.2",
       "i18next-http-backend": "^1.3.1",
       "js-cookie": "^2.2.1",
+      "plotly.js-basic-dist": "^1.58.5",
       "react": "^17.0.0",
       "react-dom": "^17.0.0",
       "react-i18next": "^11.15.3",
@@ -26,6 +27,7 @@
       "@types/react-dom": "^17.0.0",
       "@types/react-router-dom": "^5.2.0",
       "eslint-config-airbnb-typescript-prettier": "^5.0.0",
+      "prettier": "^2.5.1",
       "react-scripts": "5.0.0",
       "typescript": "^4.5.4"
     },

